2-library/6core-bom/pom.xml

```diff
@@ -36,6 +36,12 @@
                 <version>0.0.1-SNAPSHOT</version>
             </dependency>
 
+            <dependency>
+                <groupId>com.6core.lib</groupId>
+                <artifactId>logging-model</artifactId>
```

Comment:
Generally word `model` points most developers to the Entities, Dtos, Domain, and so on. 

Nothing of the above will be present in this lib. Pls rename to avoid misunderstanding in future.

Reply:
I get it.